template/.github/workflows/ci.yml

```diff
@@ -170,3 +170,60 @@ jobs:
 
     - name: Run the functional test suite
       run: poetry run invoke test --suite=functional
+
+build-and-test-docker-image:
+    name: Build and Test Docker Image
+    needs: [static-analysis-checks, type-checking, run-system-tests, run-functional-tests]
+    runs-on: ubuntu-latest
+
+    services:
+      postgres:
+        image: postgres:14
+        env:
+          POSTGRES_USER: dev
+          POSTGRES_PASSWORD: dev_password
+          POSTGRES_DB: dev
+        options: >-
+          --health-cmd pg_isready
+          --health-interval 10s
+          --health-timeout 5s
+          --health-retries 5
+        ports:
+            # Allows us to use the same env config used locally by mapping to port to
+            # the test runner container.
+            - 5432:5432
+
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v4
+
+    - name: Set up Python
+      uses: actions/setup-python@v5
+      with:
+        python-version: '3.12'
+
+    - name: Install dependencies
+      run: |
+        pip install invoke
+        cp example.env .env
+
+    - name: Modify DATABASE_URL for GitHub Actions
+      run: |
+        echo 'DATABASE_URL=postgres://dev:dev_password@postgres:5432/dev'  >> .env
+        echo 'DJANGO_SETTINGS_MODULE=avocado.main.settings' >> .env
```

Comment:
<!-- Issues highlight specific problems with the subject under review. These problems can be user-facing or behind the scenes. It is strongly recommended to pair this comment with a suggestion. If you are not sure if a problem exists or not, consider leaving a question.  -->
**issue:** ‏The project name should not need to be hard coded here

I'm also not sure why you need to set the `DJANGO_SETTING_MODULE` - that should be baked into the wsgi/asgi files when you create a project from this template.

Reply:
Sorry looks like the update for placeholder of the project name was missed.

Re the `DJANGO_SETTING_MODULE` settings for some reasons when testing locally I got an import error of `[DJANGO_SETTINGS_MODULE No Module Named. So I added this to move on.

both updated in [97b9ced](https://github.com/ackama/django-template/pull/55/commits/97b9cedbc01893c77a47e32ef306788c1183dd9b)